external/CMakeLists.txt

```diff
@@ -88,13 +88,15 @@ if (NOT ${SPIRV_SKIP_TESTS})
     if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/effcee)
       # If we're configuring RE2 (via Effcee), then turn off RE2 testing.
       if (NOT TARGET re2)
-	set(RE2_BUILD_TESTING OFF)
+        set(RE2_BUILD_TESTING OFF)
       endif()
       if (MSVC)
-	# SPIRV-Tools uses the shared CRT with MSVC.  Tell Effcee to do the same.
-	set(EFFCEE_ENABLE_SHARED_CRT ON)
+        # SPIRV-Tools uses the shared CRT with MSVC.  Tell Effcee to do the same.
+        set(EFFCEE_ENABLE_SHARED_CRT ON)
       endif()
-      add_subdirectory(effcee)
+      set(EFFCEE_BUILD_SAMPLES OFF CACHE BOOL "Do not build Effcee examples")
```

Comment:
I suggest making this only take effect if the effcee target is not already configured.
Doing so will avoid breaking a build configuration that had effcee already configured at an outer layer.

I did a similar thing at line 90 by checking for the re2 target first.

Reply:
like this snippet you mean?

https://github.com/KhronosGroup/SPIRV-Tools/blob/538512e8e8980cc01ab6501cd1cbd402d54f8491/external/CMakeLists.txt#L90-L92